python_modules/dagster/dagster/_core/definitions/repository_definition/repository_data_builder.py

```diff
@@ -429,59 +431,63 @@ def _process_and_validate_target(
     coerced_graphs: Dict[str, JobDefinition],
     unresolved_jobs: Dict[str, UnresolvedAssetJobDefinition],
     jobs: Dict[str, Union[JobDefinition, Callable[[], JobDefinition]]],
-    target: Union[GraphDefinition, JobDefinition, UnresolvedAssetJobDefinition],
+    executable_def: Union[GraphDefinition, JobDefinition, UnresolvedAssetJobDefinition],
```

Comment:
I actually don't think we should double down on `ExecutableDefinition` as a name. It feels like a very "important" name when it really is a fairly low-value alias.

Reply:
I completely agree. I would like to rename it upstack, but haven't thought through the best candidates. For this PR, I think replacing `target` with `executable_def` makes sense because: (a) the "target" name here is confusing because it actually refers to the thing wrapped by a `*Target` abstraction; (b) the type signature matches `ExecutableDefinition` (I will replace the inlined union with the alias).